22:49:50 05 Apr 2024 UTC: INFO: BaseLog.py(108): Process id = 2780269
22:49:50 05 Apr 2024 UTC: WARNING: BaseLog.py(111): Config file /home/jails/uwssc/gliderjail/home/sg195/sg195.conf was not found
22:49:50 05 Apr 2024 UTC: INFO: Utils.py(1011): Basestation version: 3.0.2; QC version: 1.12
22:49:50 05 Apr 2024 UTC: INFO: Utils.py(1017): Python version 3.10.10
22:49:50 05 Apr 2024 UTC: INFO: Utils.py(1031): Numpy version 1.25.0
22:49:50 05 Apr 2024 UTC: INFO: Utils.py(1046): Scipy version 1.10.1
22:49:50 05 Apr 2024 UTC: INFO: Utils.py(1066): seawater version 3.3.4
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1474): Invoked with command line [/usr/local/basestation3/Base.py --mission_dir /home/jails/uwssc/gliderjail/home/sg195 --verbose --make_mission_timeseries --daemon --nice 10 --base_log /home/jails/uwssc/gliderjail/home/sg195/baselog_240405224948 --web_file_location=http://iop.apl.washington.edu/seaglider/sg195/current/data --reply_addr=gbs2@uw.edu --config /home/jails/uwssc/gliderjail/home/sg195/sg195.conf --delete_upload_files --magcalfile search --auxmagcalfile search ]
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1476): PID:2780633
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): volmax value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): rho0 value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): hd_a value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): hd_b value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): hd_c value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): abs_compress value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): therm_expan value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: WARNING: CalibConst.py(122): temp_ref value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1524): Started processing 22:49:50 05 Apr 2024 GMT
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1567): Instrument ID = 195
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /usr/local/basestation3/etc/.pagers for  lategps recov critical drift
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/sg1i1w4wpibg9m8s77e19st14w,slack,gpsddmm)
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/71kp13t63ifummqbhxzniw18go,slack,alerts)
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/5t1u7ab147dstbh5iq79h48spy,slack,critical)
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:49:50 05 Apr 2024 UTC: INFO: CommLog.py(113): prefix:dive:2 calls_made:2 call_cycle:0 
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /usr/local/basestation3/etc/.pagers
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers for  lategps recov critical drift
22:49:50 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1618): Processing comm_merged.log
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1647): Finished processing comm_merged.log
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1765): Processing seaglider selftests
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1801): No new selftests to processed
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1803): Processing pdoscmd.bat logs
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1822): No pdos logfiles found to process
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1829): Processing dive(s)
22:49:50 05 Apr 2024 UTC: INFO: Base.py(541): Processing /home/jails/uwssc/gliderjail/home/sg195/sg0002lz
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002lz.1a.x00
22:49:50 05 Apr 2024 UTC: INFO: Base.py(709): Processing /home/jails/uwssc/gliderjail/home/sg195/sg0002lz.r in process_file_group
22:49:50 05 Apr 2024 UTC: INFO: Base.py(864): Removing secrets from /home/jails/uwssc/gliderjail/home/sg195/p1950002.log
22:49:50 05 Apr 2024 UTC: INFO: Base.py(541): Processing /home/jails/uwssc/gliderjail/home/sg195/sg0002dz
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002dz.1a.x00
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002dz.1a.x01
22:49:50 05 Apr 2024 UTC: INFO: Base.py(709): Processing /home/jails/uwssc/gliderjail/home/sg195/sg0002dz.r in process_file_group
22:49:50 05 Apr 2024 UTC: WARNING: DataFiles.py(265): No handler found for columns ['wlbb2fl.sig470nm', 'wlbb2fl.sig700nm', 'wlbb2fl.sig695nm'] - using default handler
22:49:50 05 Apr 2024 UTC: INFO: Base.py(541): Processing /home/jails/uwssc/gliderjail/home/sg195/sg0002kz
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.1a.x00
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.1a.x01
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.1a.x02
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.1a.x03
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.1a.x04
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1051): Checking fragment /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.1a.x05
22:49:50 05 Apr 2024 UTC: INFO: Base.py(709): Processing /home/jails/uwssc/gliderjail/home/sg195/sg0002kz.r in process_file_group
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1862): Processed dive(s) [2]
22:49:50 05 Apr 2024 UTC: INFO: Base.py(1932): Dives to process = [2]
22:49:51 05 Apr 2024 UTC: INFO: Base.py(1961): Processing (/home/jails/uwssc/gliderjail/home/sg195/p1950002) for profiles
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(1624): Loading data from original files
22:49:51 05 Apr 2024 UTC: INFO: FlightModel.py(841): Updating FM parameters using per-dive estimations for dive 1 (1.00,1.00)
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: volmax=53322.9
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: vbdbias=67.6576
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: hd_a=0.00354813
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: hd_b=0.0112202
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: hd_c=5.7e-06
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: hd_s=-0.25
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: abs_compress=7.55657e-06
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: therm_expan=7.05e-05
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: temp_ref=15
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: glider_length=1.8
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2586): FM: rho0=1027.5
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2606): auxcompass data not present, auxPressure data not present
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2764): Not using auxcompass Not using auxPressure
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2829): Directive: correct_thermal_inertia_effects
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2829): Directive: detect_conductivity_anomalies
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2829): Directive: no_interp_gc_temperatures
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(2829): Directive: no_interp_suspect_thermal_inertia_salinities
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(1443): Not correcting Truck compass heading
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(3724): ct_type:gun-style Seabird unpumped CTD
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT temperature coefficient t_g (0.00437815) differs from $SEABIRD_T_G (0.00441787) in log file (dive 2) -- using 0.00437815.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT temperature coefficient t_h (0.000631904) differs from $SEABIRD_T_H (0.000648181) in log file (dive 2) -- using 0.000631904.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT temperature coefficient t_i (2.54281e-05) differs from $SEABIRD_T_I (2.6779e-05) in log file (dive 2) -- using 2.54281e-05.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT temperature coefficient t_j (2.9111e-06) differs from $SEABIRD_T_J (3.36672e-06) in log file (dive 2) -- using 2.9111e-06.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT conductivity coefficient c_g (-9.71354) differs from $SEABIRD_C_G (-10.0982) in log file (dive 2) -- using -9.71354.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT conductivity coefficient c_h (1.11378) differs from $SEABIRD_C_H (1.1454) in log file (dive 2) -- using 1.11378.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT conductivity coefficient c_i (-0.0016462) differs from $SEABIRD_C_I (-0.00212763) in log file (dive 2) -- using -0.0016462.
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(2434): SBECT conductivity coefficient c_j (0.000200081) differs from $SEABIRD_C_J (0.000255321) in log file (dive 2) -- using 0.000200081.
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(4778): Changed (1/303) 303 to QC_BAD because CT out of water
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(4803): CTD out of the water after climb (0.219m)
22:49:51 05 Apr 2024 UTC: INFO: QC.py(510): Changed (14/303) 3 5:8 10 13:16 19 158 288 301 to QC_PROBABLY_BAD because raw temperature spikes
22:49:51 05 Apr 2024 UTC: INFO: QC.py(592): Changed (10/303) 6:9 11 12 14 15 20 298 to QC_PROBABLY_BAD because raw conductivity spikes
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(4866): Changed (35/303) 1:35 to QC_BAD because uncorrectable 8.2m raw conductivity bubble on dive
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(4866): Changed (9/303) 294:302 to QC_BAD because uncorrectable 4.1m raw conductivity bubble on climb
22:49:51 05 Apr 2024 UTC: INFO: QC.py(247): Changed (14/303) 3 5:8 10 13:16 19 158 288 301 to QC_PROBABLY_BAD because changed raw temp implies changed raw salinity
22:49:51 05 Apr 2024 UTC: INFO: QC.py(247): Changed (45/303) 1:35 294:303 to QC_BAD because changed raw cond implies changed raw salinity
22:49:51 05 Apr 2024 UTC: INFO: QC.py(510): Changed (14/303) 3 5:8 10 13:16 19 158 288 301 to QC_INTERPOLATED because temperature spikes
22:49:51 05 Apr 2024 UTC: INFO: QC.py(592): Changed (10/303) 6:9 11 12 14 15 20 298 to QC_INTERPOLATED because conductivity spikes
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(4922): Changed (35/303) 1:35 to QC_BAD because uncorrectable 8.2m conductivity bubble on dive
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(4922): Changed (9/303) 294:302 to QC_BAD because uncorrectable 4.1m conductivity bubble on climb
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(5111): Changed (40/303) 130:169 to QC_BAD because slow apogee CT flow
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(5467): SUGGESTION: 2 interp_conductivity data_points between 289 302 % 7.1m SUSPECT unresolved strong conductivity anomaly
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(5487): Changed (85/303) 1:35 130:169 294:303 to QC_BAD because bad corrected temperature and conductivity suggests bad salinity
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(5652): Using 1-mode thermal-inertia correction.
22:49:51 05 Apr 2024 UTC: INFO: QC.py(247): Changed (85/303) 1:35 130:169 294:303 to QC_BAD because changed corrected salin implies changed speed
22:49:51 05 Apr 2024 UTC: INFO: TempSalinityVelocity.py(1083): 0: max temp_c diff: 0.01C TI time: 0.007s loop: 0.007s
22:49:51 05 Apr 2024 UTC: INFO: TempSalinityVelocity.py(1083): 1: max temp_c diff: 0.01C TI time: 0.007s loop: 0.007s
22:49:51 05 Apr 2024 UTC: INFO: TempSalinityVelocity.py(1507): TSV exiting after 1 iterations
22:49:51 05 Apr 2024 UTC: INFO: TempSalinityVelocity.py(1509): max_temp_c_diff = 0.01C
22:49:51 05 Apr 2024 UTC: INFO: TempSalinityVelocity.py(1557): Changed (1/303) 293 to QC_PROBABLY_BAD because TS bad extrapolation
22:49:51 05 Apr 2024 UTC: INFO: QC.py(247): Changed (1/303) 288 to QC_INTERPOLATED because changed corrected temp implies changed corrected salinity
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(6135): 13 (4.29%) HDM speeds are QC_BAD; 0 (0.00%) are stalled (303)
22:49:51 05 Apr 2024 UTC: INFO: QC.py(247): Changed (1/303) 288 to QC_INTERPOLATED because changed corrected temp implies changed speed
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(6345): Estimated total flight and drift time: 2955.3s (SM: 281.0s)
22:49:51 05 Apr 2024 UTC: WARNING: MakeDiveProfiles.py(6376): Substantial unmodeled flight time (28.4% of total time); DAC is bad.
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(6631): Computing QC_BAD depth-average current.
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(6712): DAC east: 1.47 cm/s DAC north: 6.84 cm/s
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(6843): Starting sensor extensions data processing
22:49:51 05 Apr 2024 UTC: INFO: aa4330_ext.py(971): Changed (303/303) 1:303 to QC_UNSAMPLED because unsampled optode oyxgen
22:49:51 05 Apr 2024 UTC: INFO: QC.py(247): Changed (85/303) 1:35 130:169 294:303 to QC_BAD because changed salinity implies changed aa4330 oxygen
22:49:51 05 Apr 2024 UTC: INFO: MakeDiveProfiles.py(6847): Finished sensor extensions data processing
22:49:51 05 Apr 2024 UTC: INFO: BaseNetCDF.py(376): /usr/local/basestation3/etc/NODC.yml does not exist - skipping
22:49:51 05 Apr 2024 UTC: WARNING: BaseNetCDF.py(3551): Missing metadata for sg_cal_use_adcppressure type should be 'i'
22:49:52 05 Apr 2024 UTC: INFO: BaseNetCDF.py(376): /usr/local/basestation3/etc/NODC.yml does not exist - skipping
22:49:52 05 Apr 2024 UTC: INFO: Base.py(2040): Backing up /home/jails/uwssc/gliderjail/home/sg195/cmdfile to /home/jails/uwssc/gliderjail/home/sg195/cmdfile.2
22:49:52 05 Apr 2024 UTC: INFO: BaseDB.py(1336): logParameterChanges db opened
22:49:52 05 Apr 2024 UTC: ERROR: could not check schema:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1032, in checkSchema
    mycon.cursor().execute(f'PRAGMA user_version = {currentSchemaVersion}')
sqlite3.OperationalError: attempt to write a readonly database

22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: ERROR: BaseDB.py(1362): attempt to write a readonly database inserting parameter changes
22:49:52 05 Apr 2024 UTC: INFO: BaseDB.py(1374): logParameterChanges db closed
22:49:52 05 Apr 2024 UTC: INFO: Utils.py(1011): Basestation version: 3.0.2; QC version: 1.12
22:49:52 05 Apr 2024 UTC: INFO: Utils.py(1017): Python version 3.10.10
22:49:52 05 Apr 2024 UTC: INFO: Utils.py(1031): Numpy version 1.25.0
22:49:52 05 Apr 2024 UTC: INFO: Utils.py(1046): Scipy version 1.10.1
22:49:52 05 Apr 2024 UTC: INFO: Utils.py(1066): seawater version 3.3.4
22:49:52 05 Apr 2024 UTC: INFO: FlightModel.py(3645): Exit is False
22:49:53 05 Apr 2024 UTC: INFO: FlightModel.py(2252): Started FM cycle 2
22:49:53 05 Apr 2024 UTC: INFO: FlightModel.py(2311): New dive 2 restarting from dive 1 (0.00354813,0.0112202)
22:49:53 05 Apr 2024 UTC: INFO: FlightModel.py(2382): dives_to_update:[2]
22:49:53 05 Apr 2024 UTC: INFO: FlightModel.py(2564): Early bias dives so far [2]
22:49:53 05 Apr 2024 UTC: INFO: FlightModel.py(2566): <Dive 2 pitch_d: 13.0 p: 127.3dbar vbias: 130.9cc (67.7cc) abs_compress=4.94681e-06 RMS=1.0202cm/s>
22:49:54 05 Apr 2024 UTC: INFO: FlightModel.py(3493):  Reprocess dive 2: (False, False, False, True, True)
22:49:54 05 Apr 2024 UTC: INFO: FlightModel.py(3494):   v: 63.23 [130.89 67.66] ac: 0.345 [4.94681e-06 7.55657e-06]
22:49:54 05 Apr 2024 UTC: INFO: FlightModel.py(3540): Reprocess dives: [2]
22:49:58 05 Apr 2024 UTC: INFO: FlightModel.py(3567): Back from Reprocess.py - see /home/jails/uwssc/gliderjail/home/sg195/flight/Reprocess_0002_1712357394.log for details
22:49:58 05 Apr 2024 UTC: INFO: FlightModel.py(3580): Dive 2 sucesfully processed
22:49:58 05 Apr 2024 UTC: INFO: FlightModel.py(2382): dives_to_update:[2]
22:49:58 05 Apr 2024 UTC: INFO: FlightModel.py(2564): Early bias dives so far [2]
22:49:58 05 Apr 2024 UTC: INFO: FlightModel.py(2566): <Dive 2 pitch_d: 13.0 p: 127.3dbar vbias: 130.9cc (130.9cc) abs_compress=4.94681e-06 RMS=1.0202cm/s>
22:49:59 05 Apr 2024 UTC: INFO: Base.py(2167): FM files updated ['/home/jails/uwssc/gliderjail/home/sg195/p1950002.nc']
22:49:59 05 Apr 2024 UTC: INFO: Base.py(1367): Extension script /home/jails/uwssc/gliderjail/home/sg195/.post_dive not found
22:49:59 05 Apr 2024 UTC: INFO: Base.py(2192): Starting netcdf load to db
22:49:59 05 Apr 2024 UTC: INFO: BaseDB.py(1088): loadDB db opened
22:49:59 05 Apr 2024 UTC: ERROR: could not check schema:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1032, in checkSchema
    mycon.cursor().execute(f'PRAGMA user_version = {currentSchemaVersion}')
sqlite3.OperationalError: attempt to write a readonly database

22:49:59 05 Apr 2024 UTC: ERROR: Failed to add /home/jails/uwssc/gliderjail/home/sg195/p1950002.nc to mission sqlite db:
Traceback (most recent call last):
  File "/usr/local/basestation3/Base.py", line 2201, in main
    BaseDB.loadDB(base_opts, ncf, run_dive_plots=False)
  File "/usr/local/basestation3/BaseDB.py", line 1098, in loadDB
    loadFileToDB(base_opts, cur, filename, con, run_dive_plots=run_dive_plots)
  File "/usr/local/basestation3/BaseDB.py", line 286, in loadFileToDB
    cur.execute(f"DELETE FROM dives WHERE dive={dive};")
sqlite3.OperationalError: attempt to write a readonly database

22:49:59 05 Apr 2024 UTC: INFO: BaseDB.py(1088): loadDB db opened
22:49:59 05 Apr 2024 UTC: ERROR: could not check schema:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1032, in checkSchema
    mycon.cursor().execute(f'PRAGMA user_version = {currentSchemaVersion}')
sqlite3.OperationalError: attempt to write a readonly database

22:49:59 05 Apr 2024 UTC: ERROR: Failed to add /home/jails/uwssc/gliderjail/home/sg195/p1950002.nc to mission sqlite db:
Traceback (most recent call last):
  File "/usr/local/basestation3/Base.py", line 2201, in main
    BaseDB.loadDB(base_opts, ncf, run_dive_plots=False)
  File "/usr/local/basestation3/BaseDB.py", line 1098, in loadDB
    loadFileToDB(base_opts, cur, filename, con, run_dive_plots=run_dive_plots)
  File "/usr/local/basestation3/BaseDB.py", line 286, in loadFileToDB
    cur.execute(f"DELETE FROM dives WHERE dive={dive};")
sqlite3.OperationalError: attempt to write a readonly database

22:49:59 05 Apr 2024 UTC: INFO: Base.py(2204): netcdf load to db done
22:49:59 05 Apr 2024 UTC: INFO: Base.py(2217): Starting per-dive plots
22:49:59 05 Apr 2024 UTC: INFO: BasePlot.py(96): plot_dives db opened
22:50:06 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(253): Starting dive_vert_vel
22:50:06 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(408): Basestation: hd_a:0.003548 hd_b:0.011220 hd_c:0.000006 mass:53.765 rho:1027.5
22:50:06 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(418): LogFile: hd_a:0.003000 hd_b:0.010000 hd_c:0.000016 mass:53.765 rho:1023.0
22:50:06 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(441): Using: hd_a:0.003548 hd_b:0.011220 hd_c:0.000006 mass:53.765 rho:1023.0
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(571): Search from -450.0 cc to 500.0 cc
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(574): vol range min:52319.25, max:53168.97, mean:52673.89
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 0 CC:-450.0 vert_v:-11.01
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 1 CC:500.0 vert_v:11.65
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 2 CC:0.0 vert_v:-0.42
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 3 CC:250.0 vert_v:6.94
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 4 CC:125.0 vert_v:3.52
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 5 CC:62.5 vert_v:1.48
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 6 CC:31.2 vert_v:0.53
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 7 CC:15.6 vert_v:0.05
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 8 CC:7.8 vert_v:-0.21
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 9 CC:11.7 vert_v:-0.07
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 10 CC:13.7 vert_v:-0.01
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 11 CC:14.6 vert_v:0.02
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(657): implied_cvbd 2942, implied_volmax 53155.3, implied_max_smcc 599.1, implied_max_maxbuoy -255.4
22:50:07 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(660): min SM_CC 316.8 based on density 1.01976 at 5.00m and 150cc to raise the antenna
22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_C_VBD - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Failed to add implied_volmax to dive 2:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1178, in addValToDB
    insertColumn(dive_num, cur, var_n, val, db_type)
  File "/usr/local/basestation3/BaseDB.py", line 142, in insertColumn
    cur.execute(f"UPDATE dives SET {col} = {val} WHERE dive={dive};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_C_VBD_GSM - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_volmax_GSM - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_max_MAX_BUOY - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_max_SM_CC - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column min_SM_CC - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: INFO: DivePitchRoll.py(84): Starting dive_pitch_roll
22:50:07 05 Apr 2024 UTC: INFO: DivePitchRoll.py(272): implied_cpitch 2972.9518166444577, implied_pitchgain 16.188305422789977
22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_C_PITCH - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: Error inserting column implied_PITCH_GAIN - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:07 05 Apr 2024 UTC: ERROR: BaseDB.py(1220): Execution failed on sql 'SELECT dive, FROM dives WHERE dive <= 2 ORDER BY dive DESC LIMIT 10': near "FROM": syntax error could not fetch ['implied_C_PITCH'] for slope calculation
22:50:08 05 Apr 2024 UTC: INFO: DivePitchRoll.py(471): c_roll_dive 2146.5317420209894, c_roll_climb 1873.0412486142798
22:50:08 05 Apr 2024 UTC: ERROR: Error inserting column implied_roll_C_ROLL_DIVE - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:08 05 Apr 2024 UTC: ERROR: Error inserting column implied_roll_C_ROLL_CLIMB - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:08 05 Apr 2024 UTC: INFO: DiveMagCal.py(170): hard magcal PQR = [-0.03,0.10,0.11], converged=1, it=4
22:50:12 05 Apr 2024 UTC: INFO: DiveMagCal.py(306): hard+soft magcal PQR = [-17.53,57.04,23.97], converged=1, it=7
22:50:12 05 Apr 2024 UTC: INFO: DiveMagCal.py(312): hard+soft magcal abc = 1.000,0.001,0.005,0.006,1.061,-0.005,0.000,-0.007,1.096
22:50:18 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(253): Starting dive_vert_vel
22:50:18 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(408): Basestation: hd_a:0.003548 hd_b:0.011220 hd_c:0.000006 mass:53.765 rho:1027.5
22:50:18 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(418): LogFile: hd_a:0.003000 hd_b:0.010000 hd_c:0.000016 mass:53.765 rho:1023.0
22:50:18 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(441): Using: hd_a:0.003548 hd_b:0.011220 hd_c:0.000006 mass:53.765 rho:1023.0
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(571): Search from -450.0 cc to 500.0 cc
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(574): vol range min:52319.25, max:53168.97, mean:52673.89
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 0 CC:-450.0 vert_v:-11.01
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 1 CC:500.0 vert_v:11.65
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 2 CC:0.0 vert_v:-0.42
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 3 CC:250.0 vert_v:6.94
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 4 CC:125.0 vert_v:3.52
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 5 CC:62.5 vert_v:1.48
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 6 CC:31.2 vert_v:0.53
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 7 CC:15.6 vert_v:0.05
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 8 CC:7.8 vert_v:-0.21
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 9 CC:11.7 vert_v:-0.07
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 10 CC:13.7 vert_v:-0.01
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(617): Bias HDM 11 CC:14.6 vert_v:0.02
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(657): implied_cvbd 2942, implied_volmax 53155.3, implied_max_smcc 599.1, implied_max_maxbuoy -255.4
22:50:19 05 Apr 2024 UTC: INFO: DiveVertVelocity.py(660): min SM_CC 316.8 based on density 1.01976 at 5.00m and 150cc to raise the antenna
22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_C_VBD - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Failed to add implied_volmax to dive 2:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1178, in addValToDB
    insertColumn(dive_num, cur, var_n, val, db_type)
  File "/usr/local/basestation3/BaseDB.py", line 142, in insertColumn
    cur.execute(f"UPDATE dives SET {col} = {val} WHERE dive={dive};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_C_VBD_GSM - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_volmax_GSM - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_max_MAX_BUOY - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_max_SM_CC - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column min_SM_CC - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: INFO: DivePitchRoll.py(84): Starting dive_pitch_roll
22:50:19 05 Apr 2024 UTC: INFO: DivePitchRoll.py(272): implied_cpitch 2972.9518166444577, implied_pitchgain 16.188305422789977
22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_C_PITCH - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: Error inserting column implied_PITCH_GAIN - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:19 05 Apr 2024 UTC: ERROR: BaseDB.py(1220): Execution failed on sql 'SELECT dive, FROM dives WHERE dive <= 2 ORDER BY dive DESC LIMIT 10': near "FROM": syntax error could not fetch ['implied_C_PITCH'] for slope calculation
22:50:20 05 Apr 2024 UTC: INFO: DivePitchRoll.py(471): c_roll_dive 2146.5317420209894, c_roll_climb 1873.0412486142798
22:50:20 05 Apr 2024 UTC: ERROR: Error inserting column implied_roll_C_ROLL_DIVE - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:20 05 Apr 2024 UTC: ERROR: Error inserting column implied_roll_C_ROLL_CLIMB - skipping:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 120, in addColumn
    cur.execute(f"ALTER TABLE dives ADD COLUMN {col} {db_type};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:20 05 Apr 2024 UTC: INFO: DiveMagCal.py(170): hard magcal PQR = [-0.03,0.10,0.11], converged=1, it=4
22:50:24 05 Apr 2024 UTC: INFO: DiveMagCal.py(306): hard+soft magcal PQR = [-17.53,57.04,23.97], converged=1, it=7
22:50:24 05 Apr 2024 UTC: INFO: DiveMagCal.py(312): hard+soft magcal abc = 1.000,0.001,0.005,0.006,1.061,-0.005,0.000,-0.007,1.096
22:50:25 05 Apr 2024 UTC: INFO: BasePlot.py(130): plot_dives db closed
22:50:25 05 Apr 2024 UTC: INFO: Base.py(2222): Per-dive plots complete
22:50:25 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1477): No .extensions file /usr/local/basestation3/etc/.extensions found
22:50:25 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1400): Starting processing on .extensions section(s):('dive',)
22:50:25 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1468): Finished processing on .extensions
22:50:25 05 Apr 2024 UTC: INFO: BaseDotFiles.py(317): No .urls file /usr/local/basestation3/etc/.urls found - skipping pass 1
22:50:25 05 Apr 2024 UTC: INFO: BaseDotFiles.py(245): Starting processing on .urls pass(1)
22:50:25 05 Apr 2024 UTC: INFO: BaseDotFiles.py(310): Finished processing on .urls pass(1)
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-475-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14147-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14164-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14161-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14167-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14152-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-1807961-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14158-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=perdive","time":1712357425.0765643}
22:50:25 05 Apr 2024 UTC: INFO: MakeMissionTimeSeries.py(231): Making mission timeseries /home/jails/uwssc/gliderjail/home/sg195/sg195_Shilshole_Shakedown_timeseries.nc from files found in /home/jails/uwssc/gliderjail/home/sg195/
22:50:25 05 Apr 2024 UTC: INFO: BaseNetCDF.py(376): /usr/local/basestation3/etc/NODC.yml does not exist - skipping
22:50:25 05 Apr 2024 UTC: INFO: BasePlot.py(156): plot_mission db opened
22:50:25 05 Apr 2024 UTC: INFO: MissionEnergy.py(98): Starting mission_energy None
22:50:25 05 Apr 2024 UTC: ERROR: Could not fetch needed columns:
Traceback (most recent call last):
  File "/usr/local/basestation3/Plotting/MissionEnergy.py", line 149, in mission_energy
    start = pd.read_sql_query(
  File "/opt/basestation/lib/python3.10/site-packages/pandas/core/indexing.py", line 1103, in __getitem__
    return self._getitem_axis(maybe_callable, axis=axis)
  File "/opt/basestation/lib/python3.10/site-packages/pandas/core/indexing.py", line 1656, in _getitem_axis
    self._validate_integer(key, axis)
  File "/opt/basestation/lib/python3.10/site-packages/pandas/core/indexing.py", line 1589, in _validate_integer
    raise IndexError("single positional indexer is out-of-bounds")
IndexError: single positional indexer is out-of-bounds

22:50:26 05 Apr 2024 UTC: INFO: MissionMotors.py(73): Starting mission_motors
22:50:26 05 Apr 2024 UTC: INFO: MissionIntSensors.py(63): Starting mission_int_sensors
22:50:27 05 Apr 2024 UTC: INFO: MissionMap.py(130): Starting mission_map
22:50:27 05 Apr 2024 UTC: ERROR: mission_map:
Traceback (most recent call last):
  File "/usr/local/basestation3/BasePlot.py", line 165, in plot_mission
    fig_list, file_list = plot_func(
  File "/usr/local/basestation3/Plotting/MissionMap.py", line 187, in mission_map
    myProj = ccrs.Orthographic(central_longitude=ctrlon, central_latitude=ctrlat)
  File "/opt/basestation/lib/python3.10/site-packages/cartopy/crs.py", line 2048, in __init__
    super().__init__(proj4_params, globe=globe)
  File "/opt/basestation/lib/python3.10/site-packages/cartopy/crs.py", line 658, in __init__
    super().__init__(*args, **kwargs)
  File "/opt/basestation/lib/python3.10/site-packages/cartopy/crs.py", line 189, in __init__
    super().__init__(self.proj4_init)
  File "/opt/basestation/lib/python3.10/site-packages/pyproj/crs/crs.py", line 348, in __init__
    self._local.crs = _CRS(self.srs)
  File "pyproj/_crs.pyx", line 2378, in pyproj._crs._CRS.__init__
pyproj.exceptions.CRSError: Invalid projection: +proj=ortho +a=6378137.0 +lon_0=nan +lat_0=nan +no_defs +type=crs: (Internal Proj Error: proj_create: invalid value for lat_0)

22:50:27 05 Apr 2024 UTC: WARNING: MissionProfiles.py(137): No dives found
22:50:27 05 Apr 2024 UTC: INFO: MissionCallStats.py(140): File types found {'std', 'sgk', 'stl', 'sgd', 'stk', 'sgl'}
22:50:27 05 Apr 2024 UTC: INFO: BasePlot.py(196): plot_mission db closed
22:50:27 05 Apr 2024 UTC: INFO: MakeKML.py(1040): Started processing 22:50:27 05 Apr 2024 GMT
22:50:28 05 Apr 2024 UTC: INFO: MakeKML.py(449): Opened /home/jails/uwssc/gliderjail/home/sg195/targets.0 for processing
22:50:28 05 Apr 2024 UTC: INFO: MakeKML.py(463): ['NORTH', 'lat=4744.5', 'lon=-12223.75', 'radius=200', 'finish=000', 'goto=SOUTH']
22:50:28 05 Apr 2024 UTC: INFO: MakeKML.py(463): ['SOUTH', 'lat=4743.0', 'lon=-12223.75', 'radius=200', 'finish=180', 'goto=NORTH']
22:50:28 05 Apr 2024 UTC: INFO: MakeKML.py(1697): Finished processing 22:50:28 05 Apr 2024 GMT
22:50:28 05 Apr 2024 UTC: INFO: MakeKML.py(1701): Run time 0.533480 seconds
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1477): No .extensions file /usr/local/basestation3/etc/.extensions found
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1400): Starting processing on .extensions section(s):('global', 'mission')
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1468): Finished processing on .extensions
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const volmax
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const rho0
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const hd_a
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const hd_b
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const hd_c
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const abs_compress
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const therm_expan
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing calib const temp_ref
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2538): Processing SBECT
22:50:28 05 Apr 2024 UTC: WARNING: Base.py(2557): ALERT:calib const volmax
    WARNING: CalibConst.py(122): volmax value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const rho0
    WARNING: CalibConst.py(122): rho0 value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const hd_a
    WARNING: CalibConst.py(122): hd_a value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const hd_b
    WARNING: CalibConst.py(122): hd_b value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const hd_c
    WARNING: CalibConst.py(122): hd_c value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const abs_compress
    WARNING: CalibConst.py(122): abs_compress value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const therm_expan
    WARNING: CalibConst.py(122): therm_expan value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const temp_ref
    WARNING: CalibConst.py(122): temp_ref value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:SBECT
    SBECT <class 'type'> coefficient(s) are mismatched between sgc and log (dive 2)!
    SBECT <class 'type'> coefficient(s) are mismatched between sgc and log (dive 2)!
Consult baselog_240405224948 for details.
22:50:28 05 Apr 2024 UTC: INFO: BaseDB.py(1169): addValToDB db opened
22:50:28 05 Apr 2024 UTC: ERROR: Failed to add alerts to dive 2:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1178, in addValToDB
    insertColumn(dive_num, cur, var_n, val, db_type)
  File "/usr/local/basestation3/BaseDB.py", line 142, in insertColumn
    cur.execute(f"UPDATE dives SET {col} = {val} WHERE dive={dive};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:28 05 Apr 2024 UTC: INFO: BaseDB.py(1198): addValToDB db closed
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2609): Processed files msg:
Processing complete as of 22:50:28 05 Apr 2024 GMT
p1950002.log
p1950002.eng
p1950002.asc
p1950002.cap
plots/dv0002_diveplot.div
plots/dv0002_diveplot.webp
plots/dv0002_cog.div
plots/dv0002_cog.webp
plots/dv0002_ctw.div
plots/dv0002_ctw.webp
plots/dv0002_aa4330.div
plots/dv0002_aa4330.webp
plots/dv0002_wlbb2fl_Chlorophyll_fluorescence.div
plots/dv0002_wlbb2fl_Chlorophyll_fluorescence.webp
plots/dv0002_wlbb2fl_backscatter.div
plots/dv0002_wlbb2fl_backscatter.webp
plots/dv0002_ctd.div
plots/dv0002_ctd.webp
plots/dv0002_raw_ctd.div
plots/dv0002_raw_ctd.webp
plots/dv0002_ts.div
plots/dv0002_ts.webp
plots/dv0002_ts_raw.div
plots/dv0002_ts_raw.webp
plots/dv0002_vert_vel.div
plots/dv0002_vert_vel.webp
plots/dv0002_pitch.div
plots/dv0002_pitch.webp
plots/dv0002_roll.div
plots/dv0002_roll.webp
plots/dv0002_roll_rate.div
plots/dv0002_roll_rate.webp
plots/dv0002_magcal.div
plots/dv0002_magcal.webp
plots/dv0002_diveplot.div
plots/dv0002_diveplot.webp
plots/dv0002_cog.div
plots/dv0002_cog.webp
plots/dv0002_ctw.div
plots/dv0002_ctw.webp
plots/dv0002_aa4330.div
plots/dv0002_aa4330.webp
plots/dv0002_wlbb2fl_Chlorophyll_fluorescence.div
plots/dv0002_wlbb2fl_Chlorophyll_fluorescence.webp
plots/dv0002_wlbb2fl_backscatter.div
plots/dv0002_wlbb2fl_backscatter.webp
plots/dv0002_ctd.div
plots/dv0002_ctd.webp
plots/dv0002_raw_ctd.div
plots/dv0002_raw_ctd.webp
plots/dv0002_ts.div
plots/dv0002_ts.webp
plots/dv0002_ts_raw.div
plots/dv0002_ts_raw.webp
plots/dv0002_vert_vel.div
plots/dv0002_vert_vel.webp
plots/dv0002_pitch.div
plots/dv0002_pitch.webp
plots/dv0002_roll.div
plots/dv0002_roll.webp
plots/dv0002_roll_rate.div
plots/dv0002_roll_rate.webp
plots/dv0002_magcal.div
plots/dv0002_magcal.webp
plots/eng_mission_volmax.div
plots/eng_mission_volmax.webp
plots/eng_pitch_roll_motors.div
plots/eng_pitch_roll_motors.webp
plots/eng_vbd_motors.div
plots/eng_vbd_motors.webp
plots/eng_vbd_effic.div
plots/eng_vbd_effic.webp
plots/eng_mission_int_sensors.div
plots/eng_mission_int_sensors.webp
plots/eng_mission_depthangle.div
plots/eng_mission_depthangle.webp
plots/eng_mission_commlog.div
plots/eng_mission_commlog.webp
plots/eng_download_stats.div
plots/eng_download_stats.webp
sg195.kmz
p1950002.nc
sg195_Shilshole_Shakedown_timeseries.nc

22:50:28 05 Apr 2024 UTC: INFO: Base.py(1367): Extension script /home/jails/uwssc/gliderjail/home/sg195/.post_mission not found
22:50:28 05 Apr 2024 UTC: INFO: BaseDB.py(1169): addValToDB db opened
22:50:28 05 Apr 2024 UTC: ERROR: Failed to add capture to dive 2:
Traceback (most recent call last):
  File "/usr/local/basestation3/BaseDB.py", line 1178, in addValToDB
    insertColumn(dive_num, cur, var_n, val, db_type)
  File "/usr/local/basestation3/BaseDB.py", line 142, in insertColumn
    cur.execute(f"UPDATE dives SET {col} = {val} WHERE dive={dive};")
sqlite3.OperationalError: attempt to write a readonly database

22:50:28 05 Apr 2024 UTC: INFO: BaseDB.py(1198): addValToDB db closed
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /usr/local/basestation3/etc/.pagers for  alerts
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/sg1i1w4wpibg9m8s77e19st14w,slack,gpsddmm)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/71kp13t63ifummqbhxzniw18go,slack,alerts)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/5t1u7ab147dstbh5iq79h48spy,slack,critical)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /usr/local/basestation3/etc/.pagers
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers for  alerts
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /usr/local/basestation3/etc/.pagers for  alerts
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/sg1i1w4wpibg9m8s77e19st14w,slack,gpsddmm)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/71kp13t63ifummqbhxzniw18go,slack,alerts)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(599): Sending ALERTS FROM PROCESSING SG195 to https://sahale.apl.washington.edu/mattermost/hooks/71kp13t63ifummqbhxzniw18go
22:50:28 05 Apr 2024 UTC: INFO: SlackPost.py(59): instrument_id:195 slack_hook_url:https://sahale.apl.washington.edu/mattermost/hooks/71kp13t63ifummqbhxzniw18go subject_line:ALERTS FROM PROCESSING SG195 message_body:ALERT:calib const volmax
    WARNING: CalibConst.py(122): volmax value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const rho0
    WARNING: CalibConst.py(122): rho0 value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const hd_a
    WARNING: CalibConst.py(122): hd_a value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const hd_b
    WARNING: CalibConst.py(122): hd_b value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const hd_c
    WARNING: CalibConst.py(122): hd_c value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const abs_compress
    WARNING: CalibConst.py(122): abs_compress value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const therm_expan
    WARNING: CalibConst.py(122): therm_expan value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:calib const temp_ref
    WARNING: CalibConst.py(122): temp_ref value ignored. v3 Flight Model does not use this value. Add '% FM_ignore' to sg_calib_constants.m to suppress this warning. (Max message count exceeded)
ALERT:SBECT
    SBECT <class 'type'> coefficient(s) are mismatched between sgc and log (dive 2)!
    SBECT <class 'type'> coefficient(s) are mismatched between sgc and log (dive 2)!
Consult baselog_240405224948 for details.
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/5t1u7ab147dstbh5iq79h48spy,slack,critical)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /usr/local/basestation3/etc/.pagers
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers for  alerts
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /usr/local/basestation3/etc/.pagers for  alerts comp
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/sg1i1w4wpibg9m8s77e19st14w,slack,gpsddmm)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/71kp13t63ifummqbhxzniw18go,slack,alerts)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(388): Processing .pagers line (https://sahale.apl.washington.edu/mattermost/hooks/5t1u7ab147dstbh5iq79h48spy,slack,critical)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(393): Using sending function slack
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /usr/local/basestation3/etc/.pagers
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(380): Starting processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers for  alerts comp
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(632): Finished processing on /home/jails/uwssc/gliderjail/home/sg195/.pagers
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(990): Starting processing on .ftp
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1009): Finished processing on .ftp
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(987): No .sftp file found - skipping .sftp processing
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1026): Starting processing on .mailer
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(1377): Finished processing on .mailer
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(317): No .urls file /usr/local/basestation3/etc/.urls found - skipping pass 2
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(245): Starting processing on .urls pass(2)
22:50:28 05 Apr 2024 UTC: INFO: BaseDotFiles.py(310): Finished processing on .urls pass(2)
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-475-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14147-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14164-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14161-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14167-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14152-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-1807961-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Utils.py(1845): notifying /tmp/sanic-14158-notify.ipc:195-urls-files:{"glider":195,"dive":2,"content":"files=all","time":1712357428.7508726}
22:50:28 05 Apr 2024 UTC: INFO: Base.py(2875): Finished processing 22:50:28 05 Apr 2024 GMT
